docroot/themes/contrib/civictheme/includes/banner.inc

```diff
@@ -44,10 +44,7 @@ function _civictheme_preprocess_block__civictheme_banner(&$variables) {
 
     if (civictheme_field_has_value($entity, 'field_c_b_blend_mode')) {
       $blend_mode = civictheme_get_field_value($entity, 'field_c_b_blend_mode', TRUE);
-      if (!array_key_exists('modifier_class', $variables)) {
-        $variables['modifier_class'] = '';
-      }
-      $variables['modifier_class'] .= ' civictheme-background--' . $blend_mode;
+      $variables['blend_mode'] = 'civictheme-background--' . $blend_mode;
```

Comment:
Cannot be added with modifier class, Needs to be added for the background image div.